src/styles/ant-vars.less

```diff
@@ -137,11 +137,14 @@ https://github.com/ant-design/ant-design/blob/master/components/style/themes/def
 @checkbox-group-item-margin-right: 8px;
 
 // Radio buttons
-@radio-button-bg           : @btn-default-bg;
-@radio-button-color        : @btn-default-color;
-@radio-button-hover-color  : #fff;
-@radio-button-active-color : #fff;
-
+@radio-button-bg: @black;
+@radio-button-checked-bg: @dark-three;
+@radio-button-color: @greyish-brown;
+@radio-button-hover-color: @primary-5;
+@radio-button-active-color: @dark-three;
+@radio-disabled-button-checked-bg: tint(@black, 90%);
+@radio-disabled-button-checked-color: @disabled-color;
+@radio-wrapper-margin-right: 8px;
```

Comment:
Did you mean to keep these changes after removing the antd radio group? I guess it doesn't hurt in case we want them for future 🤷🏻‍♀️

Reply:
yeah, I figured we'd probably use the same styling for any radio buttons